src/mac.js

```diff
@@ -132,16 +132,16 @@ class MacApp extends App {
     return this.updatePlist(base, `${this.appName} ${helperSuffix}`, identifier, name)
   }
 
-  async extendAppPlist (propsOrFilename) {
+  async extendPlist (base, propsOrFilename) {
```

Comment:
Nit(naming): I want to be specific about this mostly because I see `baseDir` a lot in the tooling, and just a `base` variable might be confusing.
```suggestion
  async extendPlist (basePlist, propsOrFilename) {
```